### DISCUSSION TITLE
fds2fitmi application

### DISCUSSION THREAD

**Discussion by Wenqian10** [firemodels/fds#11838] [General]
Hello, I have some doubts about the fds2ftmi tool application. I find this tool can work on the FE model with one type of element, while it seems not to work when there is more than one type of element in the FE model. I run the APDL script ( in the attachment) with the fds2ftmi tool, and it always shows more than one extra node in the center of the SURF152 element in the FE model. Does anyone have some suggestions about that issue?

This APDL script is changed from the original example in https://github.com/firemodels/fds/tree/master/Utilities/Structural_Interaction/fds2ftmi/examples/simple_panel_hot, so it can work with the fds2ftmi tool by using the same fds file in simple_panel_hot example. 

Thank you very much!

[simple case in APDL.txt](https://github.com/firemodels/fds/files/11497203/simple.case.in.APDL.txt)

**Comment by RIO4FDS** [comment]
Hi,

When I run your script here, there is a warning ignoring line 80. Take a
look at that.
It seems that no element type 2 is generated.

Ignoring this mistake, I could run the analysis using FDS results for the
simple_case_hot on the examples folder. I do not have access to your fds
file, but I imagine it is the same.

After run your script and ignore the warning I did:
- In Ansys: run the ansys2ftmi.geom script from
fds\Utilities\Structural_Interaction\fds2ftmi\scripts
- In cmd: run fds2ftmi with this line - fds2ftmi_win simple_panel_hot 0.05
2 0 600 2 0 0 1 simple_panel_hot_to_ansys
- In Ansys: read ftmi results in ansys with the file generated by the last
step: simple_panel_hot_to_ansys.dat
- In Ansys: run the solution
- In Ansys: export the results with the script export_data_hot.ans

The results seem to be ok for the case:
[image: image.png]

You have SURF152 elements on both sides of the panel: one side "seeing"the
fire source and one in the back with "ambient temperature". Maybe you are
seeing those nodes and thinking it is a mistake.

Let me know if you could follow the steps and run your case.

Best Regards,
Julio



On Wed, May 17, 2023 at 7:57 AM Wenqian10 ***@***.***> wrote:

> Hello, I have some doubts about the fds2ftmi tool application. I find this
> tool can work on the FE model with one type of element, while it seems not
> to work when there is more than one type of element in the FE model. I run
> the APDL script ( in the attachment) with the fds2ftmi tool, and it always
> shows more than one extra node in the center of the SURF152 element in the
> FE model. Does anyone have some suggestions about that issue?
>
> This APDL script is changed from the original example in
> https://github.com/firemodels/fds/tree/master/Utilities/Structural_Interaction/fds2ftmi/examples/simple_panel_hot,
> so it can work with the fds2ftmi tool by using the same fds file in
> simple_panel_hot example.
>
> Thank you very much!
>
> simple case in APDL.txt
> <https://github.com/firemodels/fds/files/11497203/simple.case.in.APDL.txt>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/fds/discussions/11838>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADA4T77KLUIX7IBN4XRA7RTXGSVLDANCNFSM6AAAAAAYE5E6AA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
___________________________________
Julio Cesar Goncalves da Silva, D.Sc.
Mobile: +55 21 98886-1235

**Reply by Wenqian10** [reply]
Hi,

Thank you very much,  Julio. 
Now it works.  I find that some APDL code should be changed in the fds2ftmi.geom file when using more than one type of element in ANSYS. And these APDL codes should be changed based on which type of element is used in ANSYS.

Best
Wenqian
